File: src/js/sqlite_block.js
const { gaussianCDF } = require("./gaussian"),
  sqlite3 = require("sqlite3").verbose();

try {
  let query = "",
    dbPath = __dirname + "/../../sqlite/database.db",
    conn = new sqlite3.Database(dbPath);
  conn.exec(
    "DROP TABLE IF EXISTS tb_js_block; CREATE TABLE tb_js_block(id INTEGER PRIMARY KEY " +
      "AUTOINCREMENT, z_score REAL NOT NULL, cumulative_distribution REAL NOT NULL);"
  );
  for (let i = -500; i <= 500; i++) {
    let x = i / 100,
      prob = gaussianCDF(0, 1, x);
    query += `INSERT INTO tb_js_block(z_score, cumulative_distribution) VALUES (${x}, ${prob.toFixed(
      6
    )});`;
  }
  conn.exec(query);
  conn.close();
} catch (err) {
  console.log(err.message);
}

File: src/py/sqlite_block.py
from gaussian import gaussianCDF
from os.path import dirname
from sqlite3 import connect

dbPath = dirname(__file__) + '/../../sqlite/database.db'
query = ''

try:
	conn = connect(dbPath)
	cur = conn.cursor()
	cur.executescript('''DROP TABLE IF EXISTS tb_py_block; CREATE TABLE tb_py_block(id INTEGER PRIMARY KEY
	AUTOINCREMENT, z_score REAL NOT NULL, cumulative_distribution REAL NOT NULL);''')
	conn.commit()
	for i in range(-500, 501):
		x = i / 100
		prob = gaussianCDF(0, 1, x)
		query += f'INSERT INTO tb_py_block(z_score, cumulative_distribution) VALUES ({x:.2f}, {prob:.6f});'
	cur.executescript(query)
	conn.commit()
	conn.close()
except Exception as e:
	print(e)
